python/ppml/src/bigdl/ppml/tests/test_kms.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# simple tests for kms, require pktest, just for dev
+# python3 -m pktest test_kms.py
+
+import io, os
+import pytest
+from kms.client import encrypt_buf_with_key, decrypt_buf_with_key, generate_primary_key, generate_data_key
+
+# Only for test purpose, never use it in production
+os.environ['APPID'] = "63a88858-29f6-426f-b9b7-15702bf056ac"
+os.environ['APIKEY'] = "PxiX0hduXAG76cw1JMYPJWyGBMGc0muB"
+
+encrypted_primary_key_path = ""
+encrypted_data_key_path= ""
+
+ehsm_ip = "172.168.0.226"
```

Comment:
Should we provide an ehsm service for test?
Our ehsm service is always an internal and experimental one towards developers, namely, not stable and possibly causes github action failure.
Maybe using a hard-coded dummy dataKey here rather than requesting from ehsm is better.

Reply:
A stable ehsm service which does not change its interface frequently is preferred for testing purpose.

If we use a hard-coded dummy dataKey here, how can we test the kms interface?

I am not sure whether these test files should be included in this pr or not

Maybe delete these files before merging :worried:

Reply:
I recommend modify to like this:
def test_encrypt_buf():
      ...  
      try:
           encrypt_buf_with_key(ehsmip, port, ...) # if failure throw an ehsm-specific exception
      catch e:
           fernet = Fernet(**dummy_data_key**)
           decrypted_content = fernet.decrypt(buf.getvalue())
           decrypted_buf.write(decrypted_content)